### ISSUE TITLE
Possible ps_game crash on 0x00495F30

### ISSUE THREAD

**Issue by Spotin2** [kurtekat/shaiya-episode-6#105]
So I've been testing around the files and after a few hours of the server the ps_game crashes. 
What I got is that this address 0x00495F30 makes an invalid call out of region. Furthermore after some digging, I found out (which might not be correct) that `hook::user_apply_skill()` causes the crash. I've got to mention that 375, 376, 377, 378, 379, 380 and 432 skills do not exist in my Database, not sure if that could cause an issue. I'm attaching the dmp just in case. 
[20240709_084438_ps_game.dmp](https://github.com/user-attachments/files/16169146/20240709_084438_ps_game.dmp)

**Comment by kurtekat** [comment]
Do you have the text file with the crash info?

**Comment by kurtekat** [comment]
Yeah. I'm gonna need that. The dump doesn't have enough information.

**Comment by kurtekat** [comment]
[user_apply_skill.zip](https://github.com/user-attachments/files/16169789/user_apply_skill.zip)

please tell me if the problem goes away when you use that file.

**Comment by kurtekat** [comment]
@Spotin2 

can you copy and paste the log file contents and send it to me please?

**Comment by Spotin2** [comment]
Sorry for the late reply, here it is. 

[PS_GAME__system.log__20240709_084438.log](https://github.com/user-attachments/files/16171481/PS_GAME__system.log__20240709_084438.log)

**Comment by kurtekat** [comment]
dang. i won't be able to glean anything from that either. i patched this last night. maybe try it and let me know how it goes.

**Comment by Spotin2** [comment]
Okay will do, thank you for your time.

**Comment by kurtekat** [comment]
https://github.com/kurtekat/shaiya-episode-6/commit/d88c79f85a52f96bb22a9d09f8f83f568db0c50c
